### DISCUSSION TITLE
Sorting columns like in ST3 (or Excel)

### DISCUSSION THREAD

**Discussion by Frans2222** [mytourbook/mytourbook#1406] [Q&A]
Hi all,
I've a tourlist of 12000 activities, of which a few hunderd marathons. Roughly 70% with GPS data. 

I would like to filter and sort them on elapsed time. Tried to sort on column but date/time are fixed. Tried the tourfilter on distance but ofcourse not all marathons are 42196 and I get long trainingruns as well by using a bandwith. When I usr SEARCH obviously all records show up containing the word "marathon''.

I remember from ST is was possible to combine filter and sort on random columns. There must be something in MTB I'm overlooking.
Anyone ideas? Thanks in advance.

**Comment by wolfgang-ch** [comment]
The flat tourbook view https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/show-tours/tour-book can be sorted on different columns, is that not enough?

**Comment by stvzldn** [comment]
See also my feature request https://github.com/mytourbook/mytourbook/issues/1379

**Comment by Frans2222** [comment]
Sorry for my late reaction (2 weeks in China with restricted internet). The sort option in flat tourbook view is helpful, I'd just to alter columns width, thanks Wolfgang. From there I'm using a workaround now by exporting selected records in csv to Excel and after parsing creating filters there in a pivot table.
A bit cumbersome but fits my needs.
